folium/features.py

```diff
@@ -169,7 +169,7 @@ def render(self, **kwargs):
 
 
 class GeoJson(MacroElement):
-    def __init__(self, data):
+    def __init__(self, data, style_function=None):
```

Comment:
We still need to add a slim version of the simple-spec option.  I am OK doing that in another PR.  Let's just open an issue to keep track.  (Or a new comment on the current issues we have open about the simple-spec.)